submission_rules.adoc

```diff
@@ -297,19 +297,22 @@ Training benchmark directory names must be one of  { **resnet, ssd, maskrcnn, tr
 ****** accuracy/   
 ******** mlperf_log_accuracy.json
 *** compliance_checker_log.txt
-* audit/
-** <system_desc_id>/
-*** <benchmark>/
-**** <scenario>
-***** <test_id>
-****** performance/
-******* run_1/ # 1 run for every scenario
-******** mlperf_log_summary.txt
-******** mlperf_log_detail.txt
-****** accuracy/
-******* mlperf_log_accuracy.json
-****** <verify_performance.py stdout>
-****** <verify_accuracy.py stdout>
+** compliance/
+*** <system_desc_id>/
+**** <benchmark>/
+***** <scenario>
+****** <test_id>
```

Comment:
@georgelyuan Could you list out the accepted values for this below? I think the accepted values are `TEST01` `TEST04-A` `TEST04-B` and `TEST05`.